src/schema/nmdc.yaml

```diff
@@ -307,6 +307,17 @@ classes:
     exact_mappings:
       - OBI:0000747
       - NCIT:C43412
+
+  credit association:
```

Comment:
I thought we were going to call this a `study role`. Did @turbomam or @cmungall change your mind?

Reply:
I did some renaming because some PROVO predicated mappings were repeated and I was getting confused. If you prefer any of the previous names, I could change them back. The new names mirror the PROVO predicates a little better. Ie. the `study role` wasn't really a role. It was an association structure.